### PR TITLE
Feature/breadcrumbs

### PR DESCRIPTION
**Pull request by katherinebernstein**

### CODE REVIEW

File: src/containers/entitysetsearch/EntitySetSearchResults.js [hunk 1]
```diff
@@ -36,14 +39,40 @@ export default class EntitySetSearchResults extends React.Component {
   }
 
   onRowSelect = (selectedId, selectedRow, selectedEntitySetId, selectedPropertyTypes) => {
-    this.setState({ selectedId, selectedRow, selectedEntitySetId, selectedPropertyTypes });
+    EntityDataModelApi.getEntitySet(selectedEntitySetId)
```
Comment (UnsungHero97) [inline src/containers/entitysetsearch/EntitySetSearchResults.js]: All of this should be done with redux + epics.

File: src/containers/entitysetsearch/EntitySetUserSearchResults.js [hunk 2]
```diff
@@ -32,15 +35,41 @@ export default class EntitySetUserSearchResults extends React.Component {
   }
 
   onUserSelect = (selectedId, selectedRow, selectedEntitySetId, selectedPropertyTypes) => {
-    this.setState({ selectedId, selectedRow, selectedEntitySetId, selectedPropertyTypes });
-    this.props.hidePaginationFn(true);
+    EntityDataModelApi.getEntitySet(selectedEntitySetId)
```
Comment (UnsungHero97) [inline src/containers/entitysetsearch/EntitySetUserSearchResults.js]: Same comment about using redux and epics.

File: src/utils/EntityTypeTitles.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  'sample.person2': [
+    'general.lastname',
+    'general.firstname'
+  ],
+  'sample.bookings2': [
+    'publicsafety.datebooked2'
+  ]
+};
+
+export default class EntityTypeTitles {
```
Comment (UnsungHero97) [inline src/utils/EntityTypeTitles.js]: You don't need to use a class here. Just export the functions individually.

File: src/containers/entitysetsearch/EntitySetSearchResults.js (hunk 1 above)
Comment (katherinebernstein) [inline src/containers/entitysetsearch/EntitySetSearchResults.js]: none of the entity set search stuff is set up with redux yet, so it made more sense to me for that to be all done at once. I can try to do a partial redux thing though if you think that's better

File: src/containers/entitysetsearch/EntitySetUserSearchResults.js (hunk 2 above)
Comment (katherinebernstein) [inline src/containers/entitysetsearch/EntitySetUserSearchResults.js]: same

File: src/utils/EntityTypeTitles.js (hunk 3 above)
Comment (katherinebernstein) [inline src/utils/EntityTypeTitles.js]: +1

**Pull request merged**